### DISCUSSION TITLE
Displaying ggplot2 figures via RCall?

### DISCUSSION THREAD

**Discussion by rokapre** [fonsp/Pluto.jl#1085] [Questions about using Pluto]
Hi, I am wondering how do I make a figure done in ggplot2 via RCall show up in Pluto? At the top of a cell when I run it, I just see RObject{VecSxp} but the figure itself does not render as it would in a Jupyter Notebook. Do you have to use a Julia plotting package in order for things to show up in the notebook?

Thanks

**Comment by diegozea** [comment]
Hi! I had the same question some days ago. I followed the workaround in these StackOverflow answers: https://stackoverflow.com/questions/64049836/using-ggplot2-with-julia-pluto-notebook RCall is using this code for IJulia: https://github.com/JuliaInterop/RCall.jl/blob/master/src/ijulia.jl I guess that something similar could be done for Pluto. Best,
